### PR TITLE
ddt removal in ui operating system tests

### PR DESCRIPTION
**Pull request by sthirugn**
Test results:
```sh
# nosetests tests/foreman/ui/test_operatingsys.py:OperatingSys.test_create_os_with_different_names
.
----------------------------------------------------------------------
Ran 1 test in 50.558s

OK
# nosetests tests/foreman/ui/test_operatingsys.py:OperatingSys.test_create_os_with_random_params
.
----------------------------------------------------------------------
Ran 1 test in 31.597s

OK
# nosetests tests/foreman/ui/test_operatingsys.py:OperatingSys.test_negative_create_os_invalid_name
.
----------------------------------------------------------------------
Ran 1 test in 53.988s

OK
# nosetests tests/foreman/ui/test_operatingsys.py:OperatingSys.test_negative_create_os_with_wrong_major_version
.
----------------------------------------------------------------------
Ran 1 test in 69.869s

OK
]

# nosetests tests/foreman/ui/test_operatingsys.py:OperatingSys.test_negative_create_os_with_wrong_minor_version
.
----------------------------------------------------------------------
Ran 1 test in 49.198s

OK

# nosetests tests/foreman/ui/test_operatingsys.py:OperatingSys.test_update_os_basic_params
.
----------------------------------------------------------------------
Ran 1 test in 36.440s

OK

# nosetests tests/foreman/ui/test_operatingsys.py:OperatingSys.test_negative_set_os_parameter_with_long_values
.
----------------------------------------------------------------------
Ran 1 test in 58.329s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_operatingsys.py [hunk 1]
```diff
@@ -36,103 +60,64 @@ def test_create_os_with_different_names(self, name):
...
-                minor_version=gen_string('numeric', 1),
-                os_family='Red Hat',
-                archs=['x86_64']
-            )
-            self.assertIsNotNone(self.operatingsys.wait_until_element
-                                 (common_locators['name_haserror']))
-            self.assertIsNone(self.operatingsys.search(name))
-
-    @run_only_on('sat')
-    def test_negative_create_os_with_blank_name(self):
```
Comment (sthirugn) [inline tests/foreman/ui/test_operatingsys.py]: this is merged with the test_negative_create_os_invalid_name() test

**Comment by oshtaier** [comment]
ACK

File: tests/foreman/ui/test_operatingsys.py [hunk 2]
```diff
@@ -479,12 +448,16 @@ def test_negative_set_os_parameter_with_long_values(self, param):
...
-                common_locators['common_param_error']
-            ))
+            for param in invalid_values_list(interface='ui'):
+                with self.subTest(param):
+                    try:
+                        self.operatingsys.set_os_parameter(
+                            os_name, param, param)
+                    except UIError as err:
+                        self.fail(err)
+                        self.assertIsNotNone(
```
Comment (abalakh) [inline tests/foreman/ui/test_operatingsys.py]: Wrong indentation, please move this assertion out of try/except block.

**Comment by abalakh** [comment]
Pending comment.

File: tests/foreman/ui/test_operatingsys.py (hunk 2 above)
Comment (elyezer) [inline tests/foreman/ui/test_operatingsys.py]: :+1:

Comment (sthirugn) [inline tests/foreman/ui/test_operatingsys.py]: good catch. :+1:

**Comment by sthirugn** [comment]
comment updated and rebased:
```sh
# nosetests tests/foreman/ui/test_operatingsys.py -m test_negative_set_os_parameter_with_long_values
.
----------------------------------------------------------------------
Ran 1 test in 52.141s
```
OK

**Comment by abalakh** [comment]
APT

**Comment by sthirugn** [comment]
Part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**